ds_judgements_public_ui/sass/includes/_confirmation.scss

```diff
@@ -0,0 +1,17 @@
+h1.confirmation-steps {
```

Comment:
I'd like to think a bit about the scope of these rules, and  whether they live in the right place. Are they general overrides to the gov.uk styles to be used everywhere? in which case they should probably go in the gds overrides sass. Otherwise, if they're specific styles used in these pages, we should scope them as such, and use BEM, creating a new `.transactional-licence-form-content` block (or something like that, and scoping these within them), and putting them in a sass file which contains everything for that block.

Reply:
@timcowlishaw is this confirmation page obsolete in this branch now ? Is it now suppose to be part of the actual form as step 1 as in the updates in Miro shows crossed off with post it stating first page in application form - if so shall I delete this page on the branch ?
![image](https://github.com/nationalarchives/ds-caselaw-public-ui/assets/75584408/1392df18-e946-44f0-bdc7-96473a28011a)